src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/chemplant/GregtechMTE_ChemicalPlant.java

```diff
@@ -861,28 +730,6 @@ private ItemStack findCatalyst(ItemStack[] aItemInputs, ItemStack[] aRecipeInput
         return null;
     }
 
-    private void damageCatalyst(ItemStack aStack, int parallelRecipes) {
```

Comment:
Looks like catalyst damaging behavior was lost here: https://github.com/GTNewHorizons/GTplusplus/commit/7366deae4a0e4404799ab17dfde0a7ba7d4cb54f#diff-eab063fe2565d5d46d136989dfcb7dac9a63f0e7bbba9ba08cd0dab2c0c0d93dL818
Can you add it back?

Reply:
I also realized that parallels weren't correctly set to max. Also prevented that you can do a full batch, if the catalyst only has 1 durability remaining with batch mode.